NASA-AMMOS/aerie-ui
# Global scheduling conditions

Resolves #276 
Resolves #277 

To test:
1. Create a plan and navigate into it
2. Open the "Scheduling Goals" panel
3. Click on the "new" button and create some goals (for examples: https://nasa-ammos.github.io/aerie/develop/user-guide/ui-api-guide/scheduling/scheduling-goals.html)
4. Close that tab and open the new "Scheduling Conditions" panel from the plan
5. Click on the "new" button and create some conditions (for examples: https://nasa-ammos.github.io/aerie/develop/user-guide/ui-api-guide/scheduling/scheduling-conditions.html)
    
    - NOTE: The current documentation might have some errors in the examples. Specifically there is a reference to `.lowerThan(...)` when it should be `.lessThan(...)`
6. Verify that you can enable/disable a condition
7. Attempt to schedule something and verify that the condition affects the activities being generated.
